app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,28 @@
+     <!-- Site footer -->
+     <footer class="site-footer fixed-bottom">
```

Comment:
Please change from Spaces: 2 to Spaces: 4.
This is a .html.erb file.

Also please fix the indentions for the anchor tags.
The indentions for your DIV and P tags are correct so please follow them.
![image](https://user-images.githubusercontent.com/35232348/95713398-0e0a0780-0c99-11eb-9d82-a3145fd441f7.png)